### PR TITLE
CI with Github Actions

### PR DESCRIPTION
**Pull request by DagBAndersen**
Steps:
- Build and push docker containers to Dockerhub
- Copy repo to Digital Ocean
- Deploy to the server with docker-compose

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        name: dagandersen/backend
+        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
+        password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
+        workdir: backend
+    - uses: actions/checkout@master
+    - name: Push frontend to Dockerhub
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: dagandersen/frontend
+        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
```
Comment (emilbartholdy) [inline .github/workflows/main.yml]: Shouldn't this be my user account? We can use your account instead if need be 😄

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: CI
```
Comment (emilbartholdy) [inline .github/workflows/main.yml]: Suggestion: CI is rather vague name for a workflow. What about "Build and deploy"? The name of the file should be renamed accordingly.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - uses: actions/checkout@master
+    - name: Push frontend to Dockerhub
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: dagandersen/frontend
+        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
+        password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
+        workdir: frontend
+
+    - uses: actions/checkout@v1
```
Comment (emilbartholdy) [inline .github/workflows/main.yml]: Is it on purpose you use @v1 rather that @master as before here?

Comment (DagBAndersen) [reply .github/workflows/main.yml]: I have simply copy-pasted from the template. I guess `v1` is a stable build and `master` is current state of the repo... i dont know... i dont think it matters

File: .github/workflows/main.yml (hunk 1 above)
Comment (DagBAndersen) [reply .github/workflows/main.yml]: Feel free to change it! I couldn't test it since i don't have access to your account.

Comment (DagBAndersen) [reply .github/workflows/main.yml]: Det kræver du selv opretter en front- og backend container på din dockerhub account... Den kan ikke tilsluttes din account uden de er oprettet.

**Pull request merged**